app/views/teachers/TeacherStudentView.coffee

```diff
@@ -95,11 +79,122 @@ module.exports = class TeacherStudentView extends RootView
       $("[id|='visualisation']").hide()
       $(selected).show()
 
+  calculateStandardDev: ->
+    return unless @courses.loaded and @levels.loaded and @sessions?.loaded and @levelData
+
+    @courseComparisonMap = []
+    for versionedCourse in @classroom.get('courses') ? []
+      course = _.find @courses.models, (c) => c.id is versionedCourse._id
+      numbers = []
+      studentRate = 0
+      members = 0 #this is the COUNT for our standard deviation
+      for member in @classroom.get('members')
+        NUMBER = 0
```

Comment:
Gotcha, switched to lowercase - was having trouble keeping track of all the calculation while I was writing this in pseudocode. The names are just keeping track of each step of the standard deviation formula -- can certainly be condensed but I'd wanted to verify the numbers along the way.